### DISCUSSION TITLE
Using MA for Whole Home Audio

### DISCUSSION THREAD

**Discussion by paulaltomare** [music-assistant/support#1280] [Q&A]
I'm considering MA/HA for my whole home audio system. I've read/watched just about everything I can find but can't seem to figure out how to get the audio out of HA to a multi-channel amp, I'll have about 12 zones. Seems I would need 12 devices to connect to the 12 inputs of the amp...but what are those devices?
Appreciate any ideas.
Paul

**Comment by OzGav** [comment]
Picoreplayers?

**Comment by paulaltomare** [comment]
I'll look into it, thanks.
